docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -0,0 +1,23 @@
+= James Local Mail Server Configuration
+:navtitle: Configuration
+
+ * Domain names (domainlist.xml)
+ * Database (james-database.properties)
+ * LMTP (lmtpserver.xml)
+ * POP3 (pop3server.xml)
+ * DNS (dnsservice.xml)
+ * Loggin (logback.xml)
+ * ??? (jmx.properties)
+ * ??? (recipientrewritetable.xml)
+ * ??? (extensions.properties)
```

Comment:
Allows loading of user defined guice modules to support injections of user definedcomponants in their nailets or other plugins. 

For instance, you could define a blacklist api with a jpa implementation, and use it in a mailet + administrate it via webadmin.

Reply:
So IIUC, the Mailet part already has its own extension mechanism, but the webadmin part does not, so this extension mechanism would be for the webadmin part of the extension?

Reply:
`but the webadmin part does not` webadmin allow loading of custom routes actually

Reply:
Ok... I'll get back to this part in more detail another time. :-)